manual/docker/Enterprise-Edition/Deploy SeaTable-EE with Docker.md

```diff
@@ -162,15 +157,18 @@ $tree /opt/seatable/seatable-data -L 2
 │   ├── seahub-data
 │   └── seatable-license.txt
 └── ssl
-    ├── cert.pem
-    ├── chain.pem
-    ├── fullchain.pem
-    ├── privkey.pem
+    ├── account.conf
+    ├── ca
+    ├── http.header
+    ├── renew_cert
+    ├── SEATABLE_SERVER_HOSTNAME
+    ├── SEATABLE_SERVER_HOSTNAME.crt
+    ├── SEATABLE_SERVER_HOSTNAME.key
     └── README
 
 ```
 
-NOTE: The directory `ssl` is empty if Let's Encrypt is not used for HTTPS.
+NOTE: The directory `ssl` is empty if Let's Encrypt is not used for HTTPS. SEATABLE_SERVER_HOSTNAME substitutes for the host name used in the docker-compose.yml file.
```

Comment:
I'd expect "dokcer-compose.yml" to see as `docker-compose.yml` (backtick-quotes).